internal/cli/kraft/pkg/packager_kraftfile_unikraft.go

```diff
@@ -135,6 +135,18 @@ func (p *packagerKraftfileUnikraft) Pack(ctx context.Context, opts *PkgOptions,
 			return nil, err
 		}
 
+		labels := opts.Project.Labels()
+		if len(opts.Labels) > 0 {
+			for _, label := range opts.Labels {
+				kv := strings.SplitN(label, "=", 2)
```

Comment:
Let's switch this to:
```go
k, v, ok := strings.Cut(label, "=")
if !ok {
```

Reply:
This fails if someone writes `key==value` it will probably return `k` = `key` and `v` = `=value`

Are you sure you want this change?

Reply:
Ah good point; this is still at the CLI so prone to user error. Let's keep.